### PR TITLE
Add uuid@8.1

### PR DESCRIPTION
**Pull request by qdequippe**
---
name: 🎉 New Feature
about: Add php uuid extension
labels: enhancement

---

Related issue: https://github.com/shivammathur/homebrew-extensions/issues/2426

### Description

Add php uuid extension

### CODE REVIEW

File: Formula/uuid@8.1.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# typed: false
+# frozen_string_literal: true
+
+require File.expand_path("../Abstract/abstract-php-extension", __dir__)
+
+# Class for Imagick Extension
```
Comment (DracoBlue) [inline Formula/uuid@8.1.rb:6]: Please fix the typo :)

File: Formula/uuid@8.1.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+class UuidAT81 < AbstractPhpExtension
+  init
+  desc "Uuid PHP extension"
+  homepage "https://github.com/php/pecl-networking-uuid"
+  url "https://pecl.php.net/get/uuid-1.2.0.tgz"
+  sha256 "SHA"
```
Comment (DracoBlue) [inline Formula/uuid@8.1.rb:12]: Please add the proper sha256

File: Formula/uuid@8.1.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  head "https://github.com/php/pecl-networking-uuid.git"
+  license "PHP-3.01"
```
Comment (DracoBlue) [inline Formula/uuid@8.1.rb:14]: The license of uuid is LGPL-2.1

**Comment by shivammathur** [comment]
Added in #2436

**Pull request closed without merging**